CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Changelog
 
+## Next release
+### Features Added
+-
```

Comment:
nit: empty bullets here and below, maybe was intentional though :)

Reply:
intentional :D ...